src/allotropy/parsers/appbio_quantstudio/appbio_quantstudio_structure.py

```diff
@@ -39,7 +39,7 @@
     try_str_from_series_or_none,
 )
 
-UNDEFINED_SAMPLE_NAME = "N/A"
+NOT_APPLICABLE = "N/A"
```

Comment:
I also considered here exctracing this variable to the allotrope constants, what do you guys thing about that?

Reply:
I think that's a good idea for consistency.

Reply:
I agree 👍 as part of this change, i think it makes sense to replace all "N/A" literals with a constant.

Thanks for doing this!